docusaurus.config.js

```diff
@@ -101,10 +101,10 @@ const config = {
         debug: false,
       },
       navbar: {
-        title: "Celestia Docs",
         logo: {
           alt: "Celestia Logo",
```

Comment:
Alt text should be Celestia Docs

Reply:
I'm going to commit this. However, when I built it locally with the title included, it appeared along with the logo. I couldn't find a way to hide it if the title isn't included or is `null`

Reply:
I mean the `alt` field, not `title`

Reply:
Alt field is what shows up for eg a blind person or a browser that can't display images. So it makes more sense if the image cannot be displayed, for it to say Celestia Docs rather than Celestia Logo.

Reply:
I see, and I'm adding alt text as "Celestia Docs."

Reply:
Yes, realized this after I commented. Thanks again 😄